osbrain/tests/test_bugs.py

```diff
@@ -21,10 +21,6 @@ def inc(agent):
     agent = run_agent('a0')
     agent.set_attr(count=0)
     agent.each(0.0, inc)
-    while True:
-        count = agent.get_attr('count')
-        if count > sys.getrecursionlimit():
-            return
-        time.sleep(0.1)
-        if count == agent.get_attr('count'):
-            raise RuntimeError('Probable recursion level exceeded!')
+
+    limit = sys.getrecursionlimit()
+    assert wait_agent_condition(agent, lambda agent: agent.count > limit)
```

Comment:
I have no idea, I will try to figure it out... As of now, I can only confirm that in Windows, that test from the current master branch takes around 4.5 seconds, while it is taking around 0.9 seconds in Ubuntu in my VM, even though it has assigned very few resources...